backend/data/finalData/conditionsProcessed.json

```diff
@@ -324,7 +324,7 @@
     },
     "ANAT3411": {
         "original": "Prerequisite: ANAT1521 or a minimum mark of 55 in ANAT2111 or  ANAT2511<br/><br/>",
-        "processed": "ANAT1521 || 55GRADE in ANAT2111 || ANAT2511"
+        "processed": "ANAT1521 || a minimum mark of 55 in ANAT2111 || ANAT2511"
```

Comment:
sorry you seem to have missed running the processor first since i changed it in the ANAT branch

Reply:
Ah yep, need to run the entire pipeline (including the regexing stuff). I just ran the manual-fixes